RDoc-2184-Cloud Batch Limit Config - Added new section to article (#1508)
ravendb/docs: Documentation/5.3/Raven.Documentation.Pages/cloud/cloud-overview.markdown

> # NoSQL Cloud Service: Overview

Author: You answered in Slack.  I updated 4.2 as well, so now it will work for 4.2+.  Perhaps it's best just to get rid of the 5.3 version that I just made, as there are no differences.  They're exact duplicates, and if we get rid of this new version in 5.3, we'll have less to maintain. @reebhub?

> # NoSQL Cloud Service: Overview

Reviewer: yes, a single version would do

> # NoSQL Cloud Service: Overview

Author: done.  I put it into a new PR with only 4.2 to avoid problems with github needing to delete the 5.3 version.

4. **Add customized indexing configuration**  
   Click to select configuration and change the value.  
5. **Indexing Configuration Key**  
   Paste or select the configuration key that you want to change.
6. **Value**  
   Enter the new value for this configuration.
   * **Click Save** at the top of the interface when finished.

## Indexing

---

### Max Batch Size

The factors to consider if adjusting the default max indexing batch size:

* [Size of documents](https://ravendb.net/articles/dealing-with-large-documents-100-mb#:~:text=RavenDB%20can%20handle%20large%20documents,isn't%20a%20practical%20one.)
* [Complexity of calculations](../studio/database/indexes/indexing-performance#common-indexing-issues) that static index does
* [IOPS number](../cloud/cloud-instances#a-production-cloud-cluster) (IOPS - Input/Output Operations Per Second - can be adjusted according to your neeeds.)
  !["Find IOPS Number"](images\configuration-see-iops.png "Find IOPS Number")  
   1. **Products**  
      In the Cloud Portal, click the Products tab.
   2. **IOPS**  
      Your current IOPS number is written here.  
      It can be adjusted by clicking "Change Storage". 

> #### Default Cloud Configuration 

Reviewer: ```suggestion
#### Default RavenDB Cloud Configuration 
```

The default configuration sets batch sizes with the following adjustable formula:  

> `Indexing.MapBatchSize = max(Bits.PowerOf2(iops * 5), 1024);`

Author: Done

* Explanation of configuration value:

>     * `max` = the larger of the two following options:

Reviewer: we usually use parens to indicate that's a function 
```suggestion
    * `max()` = returns the larger of two arguments
```

>     * `Bits.PowerOf2(iops * 5)` = the power of two that's larger than (machine's iops speed * 5)

Reviewer: ```suggestion
    * `powerOf2()` = returns a number being a power of two that's larger than the argument passed
```

Author: Done 
`returns a number which is the power of two that's larger than the argument passed`

>       e.g. in a machine with iops of 500...500 * 2 = 2,500, so the next power of two is 2 ** 12, which is 4,096 documents.  

Reviewer: Usually `^` is used for power operator.
